### ISSUE TITLE
Problem with tooltypes and preferences for MilkyTrack 1.03

### ISSUE THREAD

**Issue by Tallguy297** [AmigaPorts/MilkyTracker#51]
Two issues...

- Preferences do not seem to get used.
When opening MilkyTracker, the setup window always appears regardless of what resolution I save in the preferences.

- There only seems to be 2 tooltypes "-bpp' and "-fullscreen'.
When opening MilkyTracker, these do not seem to do anything.

Is there something I am missing??
Thank you...

**Comment by titanlab** [comment]
@Tallguy297 Yes these are issues, no you're not missing anything - we're working on it. Thanks!
